include/dt-bindings/clock/imx_ccm.h

```diff
@@ -20,4 +20,21 @@
 #define IMX_CCM_CAN_CLK			10
 #define IMX_CCM_GPT_CLK			11
 
+#define IMX_CCM_SAI1_CLK        12
+#define IMX_CCM_SAI2_CLK        13
+#define IMX_CCM_SAI3_CLK        14
+
+#define IMX_CCM_CS1CDR_PREDIV   30
+#define IMX_CCM_CS1CDR_PODF     31
+#define IMX_CCM_CS2CDR_PREDIV   32
+#define IMX_CCM_CS2CDR_PODF     33
+
```

Comment:
`PREDIV` and `PODF` can be deleted

Reply:
as this parameter is related to SAI, leave it here for futer use?

Reply:
The above 4 (30-33) can be deleted as they are not used

Reply:
right, remove them